starter-code/index.html

```diff
@@ -20,65 +20,69 @@ <h1>Pizza Builder</h1>
     <div class="panel controls">
       <ul>
         <li v-for="ingredient in ingredients">
+
           <!-- The button must have the `active` class if the ingredient is selected -->
           <!-- When the button is clicked, it should toggle the `selected` property of the corresponding ingredient object -->
-          <button
-            :class="[
+          <button @click="ingredient.selected=!ingredient.selected" :class="[
               'btn',
-              'btn-' + ingredient.name.replace(' ', '-')
-            ]"
-          >
+              'btn-' + ingredient.name.replace(' ', '-'),
+              {active : ingredient.selected}
+            ]">
             <!--
               Define a `capitalize` filter that will take a string as an argument
               and return it with the first letter in uppercase.
               Then use it to format the name of the ingredient.
             -->
+            {{ingredient.name | capitalize}}
           </button>
         </li>
       </ul>
     </div>
     <!-- End Controls -->
 
+
+
+
     <!-- Price -->
     <aside class="panel price">
       <h2>Your pizza's price</h2>
-      <b><!-- Put the base price of the pizza here --> cheese pizza</b>
+      <b>
+        <!-- Put the base price of the pizza here -->cheese pizza</b>
       <ul>
+        <li v-for="ingredient in ingredients" v-if="ingredient.selected===true"> {{ingredient.price + " " + ingredient.name}}</li>
```

Comment:
An even better one here would be to create a computed property called `selectedIngredients` which will return a new array of only the selected ingredients. Then you can iterate over that array:
`v-for="ing in selectedIngredients"`